Makefile

```diff
@@ -204,11 +221,15 @@ _console-machine:
 
 .PHONY: console-machine01
 console-machine01:
-	@$(MAKE) --no-print-directory _console-machine	CONSOLE_PORT=4000
+	@$(MAKE) --no-print-directory _console-machine	CONSOLE_PORT=4001
 
 .PHONY: console-machine02
 console-machine02:
-	@$(MAKE) --no-print-directory _console-machine	CONSOLE_PORT=4001
+	@$(MAKE) --no-print-directory _console-machine	CONSOLE_PORT=4002
+
+.PHONY: console-machine0%
+console-machine0%:
+	@$(MAKE) --no-print-directory _console-machine	CONSOLE_PORT=400$*
```

Comment:
Can the target `console-machine0%` replace the targets `console-machine01` and `console-machine02`?

Reply:
Fixes.